clients/python/girder_client/__init__.py

```diff
@@ -168,8 +168,29 @@ class GirderClient(object):
 
     DEFAULT_API_ROOT = 'api/v1'
     DEFAULT_HOST = 'localhost'
-    DEFAULT_PORT = 443
-    DEFAULT_SCHEME = 'https'
+    DEFAULT_HTTP_PORT = 8080
```

Comment:
80 truly is the "default http port" right? Does Girder use 8080 by default right now?